website/pages/docs/secrets/databases/oracle.mdx

```diff
@@ -118,8 +118,8 @@ the proper permission, it can generate credentials.
     lease_id           database/creds/my-role/2f6a614c-4aa2-7b19-24b9-ad944a8d4de6
     lease_duration     1h
     lease_renewable    true
-    password           8cab931c-d62e-a73d-60d3-5ee85139cd66
-    username           v-root-e2978cd0-
+    password           yRUSyd-vPYDg5NkU9kDg
+    username           V_VAULTUSE_MY_ROLE_SJJUK3Q8W3BKAYAN8S62_1602543009
```

Comment:
Why is the formatting changed for usernames on some of the DB's, or were the previous examples incorrect?

Reply:
The previous examples were incorrect. Most of them were copy-pasted and never a proper representation of the usernames generated.